options/options.js

```diff
@@ -15,6 +15,9 @@ $(() => {
   $('#bitbucket-login').click(e => {
     addCred(getBitbucketParam());
   });
+  $('#gitlab-login').click(e => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/5becb274ac17be98fee6e945e23cbab998e9d302/options/options.js#L18"></span>‘e’ is defined but never used.</td>
    </tr>
  </tbody>
</table>